docs/admin/admission-controllers.md

```diff
@@ -190,25 +215,28 @@ Examples of information you might put here are:
 
 In any case, the annotations are provided by the user and are not validated by Kubernetes in any way. In the future, if an annotation is determined to be widely useful, it may be promoted to a named field of ImageReviewSpec.
 
-### ServiceAccount
+### Initializers (alpha)
 
-This plug-in implements automation for [serviceAccounts](/docs/user-guide/service-accounts).
-We strongly recommend using this plug-in if you intend to make use of Kubernetes `ServiceAccount` objects.
+This plug-in is related to the [Dynamic Admission Control](/docs/admin/extensible-admission-controllers)
+introduced in v1.7.
+The plug-in determines the initializers of a resource based on the existing
+`InitializerConfiguration`s. It sets the pending initializers by modifying the
+metadata of the resource to be created and and then blocks the resource create
```

Comment:
`and and then blocks the resource create request until all pending initializers have completed` is not very accurate. The apiserver blocks the CREATE call for 30s and then returns a 504. I think it's fine if we end the description at `by modifying the metadata of the resource`. Readers can refer to the linked doc for details.

Reply:
okay.